test/setup.js

```diff
@@ -12,36 +12,80 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import should from 'should';
 import sinon from 'sinon';
-import 'should-sinon';
+import chai, { Assertion } from 'chai';
+import 'chai/register-should';
+import chaiAsPromised from 'chai-as-promised';
+import sinonChai from 'sinon-chai';
 import naclFactory from 'js-nacl';
 
 process.env.NODE_ENV = 'test';
 
-should.use((_, Assertion) => {
-	Assertion.add('hexString', function hexString() {
-		this.params = {
-			operator: 'to be hex string',
-		};
-		Buffer.from(this.obj, 'hex')
-			.toString('hex')
-			.should.equal(this.obj);
-	});
+Assertion.addMethod('hexString', function handleAssert(type) {
+	// eslint-disable-next-line no-underscore-dangle
```

Comment:
It might be preferable to turn this off for the whole function block, or even the whole file.

Reply:
Changing to block disable